src/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -70,15 +70,18 @@ enum RequisitionStatus {
         [CREATED, EDITING, VERIFYING, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED]
     }
 
+    // Default options for outbound list, without supporting request approval
     static listOutboundOptions() {
         [CREATED, EDITING, VERIFYING, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
+    // Options for outbounds when we are supporting request approval (Added approved)
     static listOutboundOptionsWhenSupportingRequestApproval() {
         [CREATED, EDITING, APPROVED, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
-    static listRequisitionOptions() {
+    // Options for request list when we are supporting request approval (Added approved and waiting for approval)
+    static listRequestOptionsWhenSupportingRequestApproval() {
         [CREATED, EDITING, APPROVED, WAITING_FOR_APPROVAL, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
```

Comment:
looking at this stuff, I've now started to think if we should not refactor those statuses methods to:

- create a common list method that would contain the intersection of all the methods
- for particular methods just concat it with the statuses that we are adding, e.g: `listCommon().concat([APPROVED])`
- or filter it if we want to exclude something

don't know if it should be part of this ticket, but I think it would improve the readability of those methods.

Reply:
Good point, I think we can even implement some kind of a `RequisitionStatusFactory` to generate a list of the statuses based on some given parameters.
But for now, unless anyone disagrees I think we can proceed with the individual lists